File: GamerGeeksCommunity-Mobile/src/entities/Plan.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package entities;

import java.sql.Date;
import java.sql.Time;

/**
 *
 * @author msi
 */
public class Plan {
    int idPlan,idStreamer,idEvennement;
    Date date;
    Time heure;
    float duree;
    String Description;

    public Plan() {
    }

    public Plan(int idPlan, int idStreamer, Date date, Time heure, float duree, String Description, int idEvennement) {
        this.idPlan = idPlan;
        this.idStreamer = idStreamer;
        this.idEvennement = idEvennement;
        this.date = date;
        this.heure = heure;
        this.duree = duree;
        this.Description = Description;
    }

    public int getIdPlan() {
        return idPlan;
    }

    public void setIdPlan(int idPlan) {
        this.idPlan = idPlan;
    }

    public int getIdStreamer() {
        return idStreamer;
    }

    public void setIdStreamer(int idStreamer) {
        this.idStreamer = idStreamer;
    }

    public int getIdEvennement() {
        return idEvennement;
    }

    public void setIdEvennement(int idEvennement) {
        this.idEvennement = idEvennement;
    }

    public Date getDate() {
        return date;
    }

    public void setDate(Date date) {
        this.date = date;
    }

    public Time getHeure() {
        return heure;
    }

    public void setHeure(Time heure) {
        this.heure = heure;
    }

    public float getDuree() {
        return duree;
    }

    public void setDuree(float duree) {
        this.duree = duree;
    }

    public String getDescription() {
        return Description;
    }

    public void setDescription(String Description) {
        this.Description = Description;
    }

    @Override
    public String toString() {
        return "Plan{" + "idPlan=" + idPlan + ", idStreamer=" + idStreamer + ", idEvennement=" + idEvennement + ", date=" + date + ", heure=" + heure + ", duree=" + duree + ", Description=" + Description + '}';
    }
    
}

File: GGC-WEB/src/Entity/Plan.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Plan
 *
 * @ORM\Table(name="plan", indexes={@ORM\Index(name="fk_plan_streamer", columns={"idStreamer"}), @ORM\Index(name="fk_plan_evenement", columns={"idEvenement"})})
 * @ORM\Entity(repositoryClass="App\Repository\PlanRepository")
 */
class Plan
{
    /**
     * @var int
     *
     * @ORM\Column(name="idPlan", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="NONE")
     */
    private $idplan;

    /**
     * @var int
     *
     * @ORM\Column(name="idStreamer", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="NONE")
     */
    private $idstreamer;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="date", type="date", nullable=false)
     */
    private $date;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="heure", type="time", nullable=false)
     */
    private $heure;

    /**
     * @var float
     *
     * @ORM\Column(name="duree", type="float", precision=10, scale=0, nullable=false)
     */
    private $duree;

    /**
     * @var string
     *
     * @ORM\Column(name="description", type="string", length=100, nullable=false)
     */
    private $description;

    /**
     * @var int
     *
     * @ORM\Column(name="idEvenement", type="integer", nullable=false)
     */
    private $idevenement;

    public function getIdplan(): ?int
    {
        return $this->idplan;
    }

    public function getIdstreamer(): ?int
    {
        return $this->idstreamer;
    }

    public function getDate(): ?\DateTimeInterface
    {
        return $this->date;
    }

    public function setDate(\DateTimeInterface $date): self
    {
        $this->date = $date;

        return $this;
    }

    public function getHeure(): ?\DateTimeInterface
    {
        return $this->heure;
    }

    public function setHeure(\DateTimeInterface $heure): self
    {
        $this->heure = $heure;

        return $this;
    }

    public function getDuree(): ?float
    {
        return $this->duree;
    }

    public function setDuree(float $duree): self
    {
        $this->duree = $duree;

        return $this;
    }

    public function getDescription(): ?string
    {
        return $this->description;
    }

    public function setDescription(string $description): self
    {
        $this->description = $description;

        return $this;
    }

    public function getIdevenement(): ?int
    {
        return $this->idevenement;
    }

    public function setIdevenement(int $idevenement): self
    {
        $this->idevenement = $idevenement;

        return $this;
    }


}
